lib/tapioca/dsl/compilers/graphql_input_object.rb

```diff
@@ -2,6 +2,7 @@
 # frozen_string_literal: true
 
 begin
+  gem("graphql", ">= 1.13")
```

Comment:
Do we have a way of distinguishing between a load error because the gem was absent versus a load error because the gem version was unsatisfied? The only approach I can think of is to look at `Gem.loaded_specs`... 🤔

Reply:
I don't think we have, so I think we will have to leave the warning out.

Reply:
Okay, turns out that doesn't work either, as there are a couple problems:

1. Tapioca loads the compilers **after** loading the host app, so by this point we get a `Gem::LoadError` saying the gem has already been activated with a different version, rather than getting `Gem::MissingSpecVersionError`
1. Tapioca goes out of its way to silence warnings anyway
1. If we could get warnings to show, they'd show multiple times (once for each time we try to load the gem)

Based on this, I think either

- we forgo warnings entirely, or
- we check compatibility up front **prior** to loading the host app, and emit warnings there